src/libs/API.js

```diff
@@ -524,6 +524,13 @@ function SetPassword(parameters) {
     return request(commandName, parameters);
 }
 
+/**
+ * @returns {Promise}
+ */
+function User_GetBetas() {
+    return request('User_GetBetas', {});
```

Comment:
I don't think you need `{}`. The doc for `request()` says it's optional. If not passing it causes an error, I say we fix the error.  Once you remove it, can you please also remove it from `GetRequestCountryCode()`?

Reply:
Docs always lie... I'll fix this by adding a default to it